concent_api/core/tests/test_unit_validation.py

```diff
@@ -403,3 +406,42 @@ def test_that_other_messages_than_task_to_compute_causes_message_validation_erro
         with pytest.raises(ConcentValidationError) as exception_wrapper:
             validate_task_to_compute(wrong_message)
         assert_that(exception_wrapper.value.error_code).is_equal_to(ErrorCode.MESSAGE_INVALID)
+
+
+class TestValidateForcePaymentContainCorrectMessages():
```

Comment:
I think there is missing test that element on `subtask_results_accepted_list` is something different than `SubtaskResultsAccepted`.